### PR TITLE
318 Sites only filtered vcf then annotate wdl

### PR DESCRIPTION
**Pull request by RoriCremer**

### CODE REVIEW

**Review by rsasch — COMMENTED** [review]
It looks like there are a lot of comments/TODOs in `GvsSitesOnlyVCF` that are either outdated or should be documented in separate tickets.

Note: don't forget to remove your inclusion of your branch (`rc-318-sites-only-filtered-vcf-then-annotate-wdl`) in `.dockstore.yml ` before merging.

File: .dockstore.yml [hunk 1]
```diff
@@ -93,6 +93,15 @@ workflows:
...
          - master
          - ah_var_store
+   - name: GvsSitesOnlyVCF
+     subclass: WDL
+     primaryDescriptorPath: /scripts/variantstore/wdl/GvsSitesOnlyVCF.wdl
+     testParameterFiles:
+       - /scripts/variantstore/wdl/GvsSitesOnlyVCF.example.inputs.json
+     filters:
+       branches:
+         - rc-318-sites-only-filtered-vcf-then-annotate-wdl
```
Comment (RoriCremer) [inline .dockstore.yml:103]: swap this to ah_var_store

**Review by rsasch — COMMENTED** [review]
Some questions comments on `GvsSitesOnlyVCF.wdl`

**Review by rsasch — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**